docs/reference/api/openmm_solvation_afe.rst

```diff
@@ -0,0 +1,146 @@
+OpenMM Absolute Solvation Free Energy Protocol
```

Comment:
It might be worth figuring out if we're documenting things in the rst or the py files.  e.g. see #570 which I think leans heavily in favour of the rst just being stubs

Reply:
:/ that PR does a lot of changes, we probably should merge it first and then fix things here? (or the inverse)